flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/SourceQueryOperation.java

```diff
@@ -43,6 +41,7 @@
 @Internal
 public class SourceQueryOperation implements QueryOperation {
 
+    private final String INPUT_ALIAS = "$$T_SOURCE";
```

Comment:
```suggestion
    private static final String INPUT_ALIAS = "$$T_SOURCE";
```
i guess the reason why checkstyle is complaining